Src/Test.UnitTests.Sarif.PatternMatcher.Cli/AnalyzeCommandTests.cs

```diff
@@ -277,6 +280,98 @@ public void AnalyzeCommand_ShouldThrowExceptionForConflictingFileSizeOptions()
             Program.InstantiatedAnalyzeCommand.Should().NotBeNull();
         }
 
+
+        [Fact]
+        public void AnalyzeCommand_FailureLevelShouldBeDefaultUnlessDynamicallyValidated()
+        {
+            var sb = new StringBuilder();
+
+            var testCases = new[] {
+                new {
+                    jsonLevel = FailureLevel.Note,
```

Comment:
>jsonLevel [](http://example.com/codeflow?start=20&length=9)

OK, reading the rest of the test code, I get it. Try the following:

'jsonLevel': 'failureLevelConfiguredInDefinitionsJson'
'dynamicValidationEnabled' -> 'dynamicValidationEnabledOnCommandLine'
'expectedNoValidation' -> 'expectedFailureLevelIfNoValidatorsExist'
'expectedStaticOnly' -> 'expectedFailureLevelIfOnlyStaticValidatorExists'
expectedStaticDynamic -> 'expectedFailureLevelIfStaticAndDynamicValidatorExists'

These are long but descriptive and precise. You can take the expedient route of creating more concise names, but only if you please author informative code comments. I note again here that I have seen very few useful code comments in the PR's coming through our pipeline. :) Try to find the right ones to author.